docs/contracts/deposit-security-module.md

```diff
@@ -251,16 +265,30 @@ function pauseDeposits(uint256 blockNumber, Signature memory sig)
 
 | Name          | Type        | Description                                          |
 | ------------- | ----------- | ---------------- |
-| `blockNumber` | `uint256`   | Block number | 
+| `blockNumber` | `uint256`   | Block number |
```

Comment:
It's obvious that this parameter is used for a require check but it's a bit unclear what this block number represents from an outsider's perspective. Perhaps we could give a slightly broader description.

Reply:
Provided a bit more details, ty!